views/view_package.dt

```diff
@@ -10,6 +10,25 @@ block title
 	- import userman.controller;
 	- auto title = "Package " ~ packageName ~ " version " ~ versionInfo["version"].get!string;
 
+	// facebook & co.
+	meta(property="og:url", content="http#{req.tls ? `s` : ``}://#{req.host}/packages/#{packageName.split(':')[0]}")
+	meta(property="og:type", content="article")
+	meta(property="og:title", content="Package #{packageName.split(':')[0]} on DUB")
+	meta(property="og:image", content=packageInfo["logo"].opt("#{req.rootDir}public/images/default-logo.png"))
+	meta(property="og:description", content=versionInfo["description"].opt!string)
+	meta(property="og:site_name", content="DUB Package Registry")
+	meta(property="og:locale", content="en_US")
+	meta(property="article:author", content=user.fullName.length ? user.fullName : user.name)
+
+	// twitter
+	meta(name="twitter:card", content="summary")
+	//- TODO: register dub twitter account
```

Comment:
I'm not sure whether it's worthwhile to create a dub registry account
BTW tweeting about every new package is probably not very helpful either.
[dlangbot](https://twitter.com/dlangbot) doesn't have many followers.